docs/utilities.md

```diff
@@ -124,4 +124,105 @@ def setup():
 
 Results in:
 
-![dist demo](img/dist_2.png)
\ No newline at end of file
+![dist demo](img/dist_2.png)
+
+### Scale (one parameter)
+
+Scales the canvas units by x horizontally and vertically. 
+
+Usage:
+
+```python
+scale(x)
+```
+**Parameters**
+
+- x: (float) The factor by which the canvas is scaled.
+
+**Example(s):**
+
+*Scale horizontal and vertical canvas units by a factor of 2*
+
+```python hl_lines="6"
+%%ignite
+def setup():
+    size(400, 400)
+    circle(100, 100, 100)
+    fill_style("red")
+    
+    # apply scale of 2, this will scale canvas units by factor of 2 horizontally and vertically
+    scale(2)
+    circle(100, 100, 100)
+```
+
+Results in:
+
+![scale demo](img/scale_1.png)
+
+### Scale (two parameters)
+
+Scales the canvas units by x horizontally and by y vertically. 
+
+Usage:
+
+```python
+scale(x,y)
+```
+**Parameters**
+
+- x: (float) The factor by which the canvas is scaled horizontally.
+- y: (float) The factor by which the canvas is scaled vertically.
+
+**Example(s):**
+
+*Scale horizontal and vertical canvas units by factors of 0.75 and 1.25 respectively*
+
+```python hl_lines="6"
+%%ignite
+def setup():
+    size(400, 400)
+    circle(100, 100, 100)
```

Comment:
This extra circle just complicates the example (remember to edit the highlight line `hl_lines` accordingly)
```suggestion
```